src/main/kotlin/org/rust/ide/annotator/RustItemsAnnotator.kt

```diff
@@ -29,14 +29,16 @@ class RustItemsAnnotator : Annotator {
     }
 
     private fun checkModDecl(holder: AnnotationHolder, modDecl: RustModDeclItemElement) {
-        if (modDecl.isPathAttributeRequired && modDecl.pathAttribute == null) {
+        val pathAttribute = modDecl.pathAttribute
+
+        if (modDecl.isPathAttributeRequired && pathAttribute == null) {
```

Comment:
Nice catch!

This will be fixed perhaps together with this PR: https://github.com/intellij-rust/intellij-rust/pull/585#discussion_r75961718